gap/Bialgebroids.gi

```diff
@@ -514,9 +514,25 @@ InstallMethod( \.,
     Add( B, b );
     
     return b;
+     
+end );
+
+InstallMethod( ObjectInAlgebroid,
+         "for an algebroid and a vertex of a quiver",
+        [ IsAlgebroid, IsVertex ],
+  function( A, v )
+    local o;
+    o := rec();
+    ObjectifyWithAttributes( o, TheTypeObjectInAlgebroid,
+            UnderlyingVertex, v
+            );
     
+    A!.Vertices.(String(v)) := o;
+    Add( A, o );
+    return o;
 end );
 
+
```

Comment:
Please remove this extra line.

Reply:
As this pull request is already merged, I will remove this in a new pull request.